spec.md

```diff
@@ -1076,6 +1076,7 @@ For success responses, the following fields are defined:
 | plan_id | string | The ID of the Service Plan from the catalog that is associated with the Service Instance. |
 | dashboard_url | string | The URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`9189kdfsk0vfnku` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the provision request, even if the service is provisioned asynchronously. |
 | parameters | object | Configuration parameters for the Service Instance. |
+| context | object | Context information provided on provisioning and/or generated by Service Brokers for the Service Instance. |
```

Comment:
I think the use of `context` here is confusing as we use `context` for metadata sent from platform to broker

Reply:
@tinygrasshopper I agree, `context` is not a good name for this and I propose something like `attributes` and I updated this PR to reflect this change 👍

Reply:
I still believe that context should be part of the response and should include the values provided by the client.